models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md

```diff
@@ -0,0 +1,76 @@
+# bert-large-uncased-whole-word-masking-qcemb-0001
+
+## Use Case and High-Level Description
+
+This is BERT-large model finetuned on SQuAD v1.1 training set from original
+bert-large-uncased-whole-word-masking provided by the [Transformers](https://github.com/huggingface/transformers) library.
+The model performs embeddings for context or question for English language;
+the input is a context or question to them,
+and the output is the 1024D embedding vectors that allow to find context with answer to the question by simple comparison the context and the question embedding vectors in the 1024D embedding space.
+For details about the original model, check out
+[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805),
+[HuggingFace's Transformers: State-of-the-art Natural Language Processing](https://arxiv.org/abs/1910.03771).
+
+Tokenization occurs using the BERT tokenizer (see the demo code for implementation details) and the enclosed `vocab.txt` dictionary file. Input is to be lower-cased before tokenizing.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 246.93 (for [1,384] input size)|
```

Comment:
```suggestion
| GOps              | 246.93                |
```
Or comment this specification.

Reply:
the model is used in 2 modes with [1,384] input shape and with [1,32] input shape. So it should be noted for which shape GOps is calculated.

Reply:
@eizamaliev what do you mean "comment this specification". can you provide example.

Reply:
@krodyush , I think @eizamaliev  means that this comment should be provided outside of the table in model description like clarification how metric was calculated (add sentence  `GOps calculated for model with input size [1, 384[` or something simmilar)